grid-extender/README.md

```diff
@@ -1,6 +1,9 @@
 # Custom grid
 
-You can add custom columns in the grid that show more information about the specific data item, for example related data or media, or any other kind of information, like data from external systems. To do this, you use a specific extensibility point. You have to develop a custom implementation of the [**columnsProvider**](http://admin-app-extensions-docs.sitefinity.site/interfaces/columnsprovider.html) interface and its method [**getColumns()**](http://admin-app-extensions-docs.sitefinity.site/interfaces/columnsprovider.html#getcolumns), which returns an **Observable** of the [**ColumnModel**](http://admin-app-extensions-docs.sitefinity.site/interfaces/columnmodel.html) object.
+You can add custom columns in the grid that show more information about the specific data item, for example related data or media, or any other kind of information, like data from external systems. What is more you can also remove some of the existing columns that bring no value to you. In order to achieve this, you have to use a specific extensibility point. A custom implementation of the [**ListColumnsProvider**](http://admin-app-extensions-docs.sitefinity.site/interfaces/listcolumnsprovider.html) interface must be provided. 
+
+Its method [**getColumnsToRemove()**](http://admin-app-extensions-docs.sitefinity.site/interfaces/listcolumnsprovider.html#getcolumnstoremove) is used to remove some of the default columns. The method returns an **Observable<string[]>** with the names of the columns to be removed.
+Its other method [**getColumns()**](http://admin-app-extensions-docs.sitefinity.site/interfaces/listcolumnsprovider.html#getcolumns) is used to add new custom columns. The method returns an **Observable** of the [**ColumnModel**](http://admin-app-extensions-docs.sitefinity.site/interfaces/columnmodel.html) object. 
```

Comment:
mb reorder so that getColumns() is first?

Reply:
done